src/esp/bindings_js/utils/WindowlessEmscriptenApplication.js

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Magnum already provides this file, so we shouldn't add another copy in the habitat-sim repo.